proteus/mprans/RANS2P.py

```diff
@@ -391,6 +400,56 @@ def attachModels(self,modelList):
         if self.epsFact_solid is None:
             self.epsFact_solid = numpy.ones(self.model.mesh.elementMaterialTypes.max()+1)
         assert len(self.epsFact_solid) > self.model.mesh.elementMaterialTypes.max(), "epsFact_solid  array is not large  enough for the materials  in this mesh; length must be greater  than largest  material type ID"
+        arbTest = True
```

Comment:
This should be refactored to allow passing in a function for phi and vof. We do this in a lot of other places. @manuel-quezada needs this for RANS3P as well.